systems/daint-mc.sh

```diff
@@ -11,14 +11,16 @@ module load CMake
 
 module load cray-hdf5 cray-netcdf
 
-# PyExtensions is needed for cython, mpi4py and others.
-# It loads cray-python/3.6.5.1
-module load PyExtensions/3.6.5.1-CrayGNU-18.08
-ns_python=$(which python3)
+module load cray-python/3.6.5.1
+ns_python=python3
 
 # load after python tools because easybuild...
 module swap gcc/7.3.0
 
+# add mpi4py to virtualenv build
```

Comment:
Should this also be added in `systems/daint-gpu.sh`?

Reply:
Yes.